### ISSUE TITLE
Capitalize one first word doesn't work

### ISSUE THREAD

**Issue by bwv656** [Benature/obsidian-text-format#11]
Thanks for writing this wonderful plug-in. I find it very helpful. However, when I try to CApitalize on the first word the following, it doesn't work. Would you please look into this? Thank you so much!

```
OH, WHY FOR US THE BLIGHTED BLOOM.
THE BLOSSOM THAT LIES WITHERING.  
THE MASTER OF LIFE'S CHANGELESS LOOM.  
HATH WROUGHT FOR US NO CHANGELESS THING.
```

![Kapture 2021-11-28 at 11 59 51](https://user-images.githubusercontent.com/10276133/143778289-9a59e2c0-9c51-4c97-b07c-cd22b844e1bf.gif)

**Comment by Benature** [comment]
fix in version 1.4.2, plz have a try.

**Comment by bwv656** [comment]
Absolutely brilliant! Works perfectly now. Thanks again for sharing this wonderful plug-in with the world!!
